crates/viewer/re_dataframe_ui/src/datafusion_table_widget.rs

```diff
@@ -481,3 +490,62 @@ impl egui_table::TableDelegate for DataFusionTableDelegate<'_> {
         re_ui::DesignTokens::table_line_height() + CELL_MARGIN.sum().y
     }
 }
+
+fn header_tooltip_ui(ui: &mut egui::Ui, column: &ColumnDescriptorRef<'_>) {
+    match column {
+        ColumnDescriptorRef::RowId(desc) => {
+            header_property_ui(ui, "Type", "row id");
+            header_property_ui(ui, "Sorted", sorted_text(desc.is_sorted));
+        }
+        ColumnDescriptorRef::Time(desc) => {
+            header_property_ui(ui, "Type", "index");
+            header_property_ui(ui, "Timeline", desc.timeline_name());
+            header_property_ui(ui, "Sorted", sorted_text(desc.is_sorted()));
+            datatype_ui(ui, &column.display_name(), desc.datatype());
+        }
+        ColumnDescriptorRef::Component(desc) => {
+            header_property_ui(ui, "Type", "component");
+            header_property_ui(ui, "Name", desc.component_name.full_name());
```

Comment:
Let's be explicit instead. "Name" is way too vague.

```suggestion
            header_property_ui(ui, "Component", desc.component_name.full_name());
```